assets/js/src/navigation.js

```diff
@@ -147,9 +148,9 @@ function initNavToggleSmall() {
 		return;
 	}
 
-	navTOGGLE.forEach( function( nav ) {
-		initEachNavToggleSmall( nav );
-	});
+	for ( let i = 0; i < navTOGGLE.length; i++ ) {
+		initEachNavToggleSmall( navTOGGLE[ i ]);
```

Comment:
Can you fix the spacing? `initEachNavToggleSmall( navTOGGLE[i] );`

Reply:
Done, but similar error reported by the build script.